### PR TITLE
Fix: Reference

### PR DESCRIPTION
**Pull request by localheinz**
This PR

* [x] fixes a reference

### CODE REVIEW

**Comment by keradus** [comment]
just double chceking - master or 2.15/2.16 branch?

**Comment by localheinz** [comment]
@keradus 

Checked, it should go into `master`, I believe!

File: src/Fixer/PhpUnit/PhpUnitOrderedCoversFixer.php [hunk 1]
```diff
@@ -19,7 +19,7 @@
 use PhpCsFixer\FixerDefinition\FixerDefinition;
 
 /**
- * @deprecated since 2.16, replaced by PhpUnitOrderedAnnotationsFixer
+ * @deprecated since 2.16, replaced by PhpDocOrderByValueFixer
```
Comment (keradus) [inline src/Fixer/PhpUnit/PhpUnitOrderedCoversFixer.php:22]: `PhpdocOrderByValueFixer` instead?

**Review by kubawerlos — APPROVED** [review]

**Comment by keradus** [comment]
Thank you @localheinz.

**Comment by localheinz** [comment]
Thank you, @keradus and @kubawerlos!

**Pull request merged**